### DISCUSSION TITLE
.client suffix not work

### DISCUSSION THREAD

**Discussion by sinasedaghat** [nuxt/nuxt#26832] [Questions]
Hello, I use the .client suffix to render the component on the client side, but when using this component, I get the error message 'Uncaught (in promise) DOMException: String contains an invalid character' and the component is not rendered.
please guide me

**Comment by IsraelOrtuno** [comment]
Would you mind to share some code, hard to debug without context. The component's code and where you are using it would be nice

**Reply by sinasedaghat** [reply]
Comments.client.vue
`
<template>
  this is Comments.client.vue component and client side render
</template>
`


app.vue
`
<template>
  <Comments />
</template>
`

**Reply by IsraelOrtuno** [reply]
Try wrapping your Comments.client.vue template into a div

https://stackblitz.com/edit/github-1ym7gy-zvkdag

**Reply by sinasedaghat** [reply]
i do but not work 
please check my code in [my repository](https://github.com/sinasedaghat/nuxt3_template/tree/use-component)

**Comment by sinasedaghat** [comment]
these component should has a wrapper
